.github/PULL_REQUEST_TEMPLATE/development.md

```diff
@@ -0,0 +1,42 @@
+## Pre-Review Checklist for the PR Author
+
+1. [ ] Branch follows the naming format (`iox-#123-this-is-a-branch`)
```

Comment:
I just noticed that we don't explicitly mention that `#123` is an issue number. Should we add that to CONTRIBUTING.md? So an issue has to be created prior to creating a branch? Just remembered as there was some confusion in PR #345 about this.

Reply:
you are right. It's not yet in CONTRIBUTION.md. I will add it